### ISSUE TITLE
Fix navbar link issue

### ISSUE THREAD

**Issue by ritesh-kanchi** [ritesh-kanchi/axya#18]
Should be a fairly easy fix. Add base_url and domain support to allow "/" to be proper for sites that aren't hosted on the root of the domain (i.e., Systems for Accessibility)
